docs/components.rst

```diff
@@ -1,12 +1,12 @@
 Components
 ==========
 
-Everything which is required to provision a virtual machine or Docker container are wrapped under components.
+Everything which is required to provision a virtual machine and Docker container are wrapped under components.
 Read further for complete details.
 
 Infrastructure
 --------------
-Infrastructure is a remote system which uses libvirt API and QEMU hypervisor installed.
+Infrastructure is a remote system which uses Libvirt API and QEMU hypervisor installed.
```

Comment:
Infrastructure is a remote system which uses libvirt API and QEMU hypervisor installed in it.

While writing libvirt API, L should be small

Reply:
Libvirt is a proper noun. Everywhere in docs Libvirt is written with **L** capital